web/api/v1/api.go

```diff
@@ -933,10 +937,12 @@ func (api *API) rules(r *http.Request) apiFuncResult {
 
 	for i, grp := range ruleGroups {
 		apiRuleGroup := &RuleGroup{
-			Name:     grp.Name(),
-			File:     grp.File(),
-			Interval: grp.Interval().Seconds(),
-			Rules:    []rule{},
+			Name:           grp.Name(),
+			File:           grp.File(),
+			Interval:       grp.Interval().Seconds(),
+			Rules:          []rule{},
+			EvaluationTime: grp.GetEvaluationDuration().Microseconds(),
```

Comment:
We keep these durations in seconds, so let's do:

```suggestion
			EvaluationTime: grp.GetEvaluationDuration().Seconds(),
```